_includes/config-guide/custom-logger-handler-examples.md

```diff
@@ -0,0 +1,157 @@
+You can use one of the following approaches for logging into custom file:
+- setup custom log file in the `di.xml`
+- setup custom file in the custom logger handler class 
+
+## Setup custom log file in the `di.xml`
+
+1. Define custom log file in the virtual type
+
+```xml
+<virtualType name="Magento\Payment\Model\Method\VirtualDebug" type="Magento\Framework\Logger\Handler\Base">
+    <arguments>
+        <argument name="fileName" xsi:type="string">/var/log/payment.log</argument>
+    </arguments>
+</virtualType>
+```
+
+1. Specify handler which will be used
+
+```xml
+<virtualType name="Magento\Payment\Model\Method\VirtualLogger" type="Magento\Framework\Logger\Monolog">
+    <arguments>
+        <argument name="handlers" xsi:type="array">
+            <item name="debug" xsi:type="object">Magento\Payment\Model\Method\VirtualDebug</item>
+        </argument>
+    </arguments>
+</virtualType>
+```
+
+1. Inject class of virtual logger to `$this->logger`
```

Comment:
Could we say that we inject the declared virtual type into our object instead of `$this->logger`? We clarify that it's accessible via $this->logger below

Reply:
**added**